optimade/filterparser/tests/test_filterparser.py

```diff
@@ -38,3 +38,166 @@ def test_repr(self):
         self.assertIsNotNone(repr(self.parser))
         self.parser.parse(self.test_filters[0])
         self.assertIsNotNone(repr(self.parser))
+
+
+class ParserTestNew(unittest.TestCase):
```

Comment:
In accordance with the earlier suggestion:
```suggestion
class ParserTestV0_10_0(unittest.TestCase):
```
Maybe also variant should be added?
Or these classes should be split into sub-files for each parser version?

Reply:
In general, I think it is perfectly fine if a "variant" has been tested only for the implementation of their `Transformer` class. I hope that in the near future there will be a single grammar file and multiple implementations of the `Transformer` classes for each variant...